app/javascript/css/colors.scss

```diff
@@ -100,9 +100,9 @@
   --input-text-color: #454545;
   --form-element-text-color: #454545;
 
-  --button-default-color: var(--primary-color);
-  --button-default-hover-color: var(--primary-hover-color);
-  --button-default-active-color: var(--primary-color-darker);
+  --button-default-color: #a21e1f;
```

Comment:
we are making all the primary buttons red?

Reply:
negative

I had made this change correctly twice before, but had to completely wipe my local repo twice as I was dealing with some git shenanigans